readme.md

```diff
@@ -66,6 +66,7 @@ $ np --help
     --no-cleanup  Skips cleanup of node_modules
     --yolo        Skips cleanup and testing
     --no-publish  Skips publishing
+    --no-draft    Don't open a GitHub release draft
```

Comment:
Should be added to the bottom. The flags are sorted by importance, not connection.

Reply:
Done both in `--help` and the README